samples/features/sql-big-data-cluster/private-bdc/scripts/deploy-private-aks-udr.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+export SUBID= <your Azure subscription ID>
```

Comment:
Could we make all these 3 variables parametrized? It's ok to have defaults, but without drawing attention to the settings that users might want to change, they might end up in a place they don't want to be. It's not suitable for all users to deploy in northeurope for example. Look here for an example: https://github.com/microsoft/sql-server-samples/blob/master/samples/features/sql-big-data-cluster/deployment/aks/deploy-sql-big-data-aks.py